nx.json

```diff
@@ -1,19 +1,80 @@
 {
   "implicitDependencies": {
-    "package.json": { "dependencies": "*", "devDependencies": "*" },
+    "package.json": {
+      "dependencies": "*",
+      "devDependencies": "*"
+    },
     ".eslintrc.json": "*"
   },
-  "affected": { "defaultBase": "main" },
+  "affected": {
+    "defaultBase": "main"
+  },
   "npmScope": "ngx-http-request-state",
   "tasksRunnerOptions": {
     "default": {
       "runner": "@nrwl/workspace/tasks-runners/default",
-      "options": { "cacheableOperations": ["build", "lint", "test", "e2e"] }
+      "options": {
+        "cacheableOperations": [
+          "build",
+          "lint",
+          "test",
+          "e2e"
+        ]
+      }
     }
   },
   "projects": {
-    "examples": { "tags": [] },
-    "examples-e2e": { "tags": [], "implicitDependencies": ["examples"] },
-    "ngx-http-request-state": { "tags": [] }
+    "examples": {
+      "tags": []
+    },
+    "examples-e2e": {
+      "tags": [],
+      "implicitDependencies": [
+        "examples"
+      ]
+    },
+    "ngx-http-request-state": {
+      "tags": []
+    }
+  },
+  "targetDependencies": {
+    "build": [
+      {
+        "target": "build",
+        "projects": "dependencies"
+      }
+    ]
+  },
+  "cli": {
+    "defaultCollection": "@nrwl/angular"
+  },
+  "defaultProject": "examples",
+  "generators": {
+    "@nrwl/angular": {
+      "application": {
+        "linter": "eslint"
+      },
+      "library": {
+        "linter": "eslint"
+      },
+      "storybook-configuration": {
+        "linter": "eslint"
+      }
+    },
+    "@nrwl/angular:application": {
+      "style": "css",
+      "linter": "eslint",
+      "unitTestRunner": "jest",
+      "e2eTestRunner": "cypress"
+    },
+    "@nrwl/angular:library": {
+      "style": "css",
+      "linter": "eslint",
+      "unitTestRunner": "jest"
+    },
+    "@nrwl/angular:component": {
+      "style": "css",
+      "changeDetection": "OnPush"
```

Comment:
Is this generated by default? If so, it's almost like a nod of acknowledgement to Angular for `OnPush` to be an implicit default.

Reply:
It's something I added in angular.json.  The nx 12 migration moved the old 'schematics' config from there to the 'generator' config in nx.json.